README.md

```diff
@@ -24,6 +24,7 @@
 	- [Configure SonarQube to use Microsoft SQL Database](#configure-sonarqube-to-use-microsoft-sql-database)
 	- [Secure the SonarQube Portal](#secure-the-sonarqube-portal)
 - [Conclusion](#conclusion)
+- [Appendix 1: Upgrading from v0.9 of the SonarQube MSBuild Runner](#upgradeFrom09Anchor)

```

Comment:
Normally we'd just be able to create a link to a header by using the text of the header. However, the markup processor doesn't like something about this header (probably the ":" character) so I defined an explicit anchor.